### PR TITLE
hardening/service_routine_doc

### PR DESCRIPTION
**Pull request by kzangeli**
Fix in service routine document and a bug fix about the `Allow:` HTTP header

### CODE REVIEW

File: src/app/contextBroker/orionRestServices.cpp [hunk 1]
```diff
@@ -401,7 +401,7 @@ static RestService badVerbV[] =
   { BatchQueryRequest,                             3, { "v2", "op", "query"                                                            }, badVerbPostOnly           },
   { BatchUpdateRequest,                            3, { "v2", "op", "update"                                                           }, badVerbPostOnly           },
   { RegistrationRequest,                           3, { "v2", "registrations", "*"                                                     }, badVerbGetDeleteOnly      },
-  { RegistrationsRequest,                          2, { "v2", "registrations"                                                          }, badVerbPostOnly           },
+  { RegistrationsRequest,                          2, { "v2", "registrations"                                                          }, badVerbGetPostOnly        },
```
Comment (fgalan) [inline src/app/contextBroker/orionRestServices.cpp]: Should the bugfix be mentioned in CNR (without referrencing issue number)? Or is too minor?

Comment (kzangeli) [reply src/app/contextBroker/orionRestServices.cpp]: I'd say it's very minor, but I have no problem with adding it to the change log.

Comment (fgalan) [reply src/app/contextBroker/orionRestServices.cpp]: Ok. Add it, pls.

Comment (kzangeli) [reply src/app/contextBroker/orionRestServices.cpp]: Fixed in cb6054704

**Pull request merged**